### PR TITLE
Added Northern Ireland geography and area selection AND corresponding jitter plots

### PR DESCRIPTION
**Pull request by MatteoLarrode**
Hey Mike, as stated in the title of the PR, I made both geographies of Northern Ireland available to the user of the HIE. 

I also made changes to jitterPlot.R to enable the creation of the Northern Ireland jitter plots. 

You will notice that I did not add `northern_ireland_ltla_shp` and `northern_ireland_hsct_shp` to data/. 
That is because Niuni will do it, along with adding the mapping of Northern Ireland to the HIE.

### CODE REVIEW

File: R/jitterPlot.R [hunk 1]
```diff
@@ -37,6 +37,20 @@ jitterPlotServer <- function(id, selected, type) {
...
           stop("No data selected", call. = FALSE)
         )
+      } else if (selected$geography == "northern_ireland_ltla_shp") {
+        switch(type,
+          "summary_metrics" = northern_ireland_ltla_summary_metrics,
+          "secondary_care" = northern_ireland_ltla_secondary_care,
+          "demographics" = northern_ireland_ltla_demographics,
+          stop("No data selected", call. = FALSE)
+        )
+      }else if (selected$geography == "northern_ireland_hsct_shp") {
```
Comment (MikeJohnPage) [inline R/jitterPlot.R:47]: Please style the file with `styler` to add the appropriate space.

**Review by MikeJohnPage — CHANGES_REQUESTED** [review]
As part of this pull request please can we add the relevant indicator descriptions in `R/indicatorDescriptions.R` to tie everything together nicely?

**Comment by MikeJohnPage** [comment]
Everything else looks great.

**Comment by MatteoLarrode** [comment]
I had forgotten about indicator descriptions! I added them now. I kept the same structure as that of the other nations, but made sure to include the links Niuni and I used for the data.

**Comment by MatteoLarrode** [comment]
I also fixed some typos I found in the file, and styled it.

**Comment by MikeJohnPage** [comment]
All looks good. Thanks, merging now 🚀

**Pull request merged**